build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/BwcSetupExtension.java

```diff
@@ -83,6 +83,11 @@ private TaskProvider<LoggedExec> createRunBwcGradleTask(
                 return getJavaHome(Integer.parseInt(minimumCompilerVersion));
             }));
 
+            // temporally workaround for reworked gradle enterprise setup
+            // removed when PR https://github.com/elastic/elasticsearch/pull/102180 backported
+            // to all BWC branches
+            loggedExec.getEnvironment().put("BUILDKITE", "false");
```

Comment:
The init script is still properly applying to nested builds though, yes?

Reply:
I tested the documented init scripts and setup. It should still work with this change